src/v/kafka/client/client.h

```diff
@@ -133,6 +133,8 @@ class client {
 
     ss::future<> update_metadata() { return _wait_or_start_update_metadata(); }
 
+    ss::future<bool> is_connected() const { return _brokers.empty(); }
```

Comment:
can you explain this one? it doesn't really read intuitively.

Reply:
That is a bug, must be negated